ldcbench.http-node/src/main/java/org/dice_research/ldcbench/nodes/http/simple/dump/comp/TarArchiver.java

```diff
@@ -0,0 +1,53 @@
+package org.dice_research.ldcbench.nodes.http.simple.dump.comp;
+
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileOutputStream;
+import org.apache.commons.compress.archivers.tar.TarArchiveOutputStream;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TarArchiver extends Archiver {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(TarArchiver.class);
+
+    private static final String TAR_MEDIA_TYPE = "application/x-tar";
+    private static final String TAR_FILE_NAME_EXTENSION = ".tar";
+    CompressionStreamFactory compressionFactory = null;
```

Comment:
Class attributes should have an explicit visibility defined. I would suggest `private` or `protected`.